graphw00f/helpers.py

```diff
@@ -255,7 +255,13 @@ def get_engines():
       'url':'https://github.com/appointy/jaal',
       'ref':'https://github.com/nicholasaleks/graphql-threat-matrix/blob/master/implementations/jaal',
       'technology':['Golang']
-    }
+    },
+    'absinthe-graphql':{
+      'name':'absinthe-graphql',
+      'url':'https://github.com/absinthe-graphql/absinthe',
+      'ref':'https://github.com/nicholasaleks/graphql-threat-matrix/blob/master/implementations/absinthe-graphql.md',
```

Comment:
Ideally this should exist as well (separate PR) so enumeration of absinthe can also detail what security features it has or doesn't have

Reply:
@dolevf So should I create a separate PR with these modifications?

Reply:
yep, please

Reply:
Done, please take a look: https://github.com/dolevf/graphw00f/pull/31